packages/spectacle/src/components/slide-layout.tsx

```diff
@@ -294,9 +319,264 @@ const MultiCodeLayout = ({
   );
 };
 
+/**
+ * Generic styled-component Image utilities
+ */
+const Img = styled.img<{ objectFit?: string }>`
+  min-width: 100%;
+  min-height: 100%;
+  max-width: 100%;
+  object-fit: ${(props) => props.objectFit || 'contain'};
+`;
+
+const ImgContainer = styled(FlexBox)`
+  overflow: hidden;
+`;
+
+const Image = ({
+  src,
+  alt = '',
+  imgContainerProps,
+  imgProps,
+  objectFit
+}: {
+  src: string;
+  alt: string;
+  imgContainerProps?: ComponentProps<typeof FlexBox>;
+  imgProps?: React.ImgHTMLAttributes<HTMLImageElement>;
+  objectFit?: string;
```

Comment:
You can get a bit more precise type by using something like `React.CSSProperties['objectFit']` instead of `string`.